cmd/commands/function.go

```diff
@@ -188,6 +189,13 @@ From then on you can use the sub-commands for the 'service' command to interact
 		"verbose", "v", verboseUsage,
 	).NoOptDefVal = "true"
 
+	command.Flags().VarPF(
+		BroadcastBoolValue(false,
```

Comment:
A broadcast value only makes sense if it is applied to several variables. This can be simplified to a more traditional cobra flag

Reply:
@ericbottard Fair enough, but I was copying the pattern employed elsewhere. I would prefer to raise an issue to apply the simplification throughout rather than "step out of line" in this PR.